app/assets/javascripts/oxalis/api/api_latest.js

```diff
@@ -243,6 +245,31 @@ class TracingApi {
     });
   }
 
+  /**
+   * Makes the specified tree active. Within the tree, the node with the highest ID will be activated.
+   *
+   * @example
+   * api.tracing.setActiveTree(3);
+   */
+  setActiveTree(treeId: number) {
+    const tracing = Store.getState().tracing;
+    assertSkeleton(tracing);
+    Store.dispatch(setActiveTreeAction(treeId));
+  }
+
+  /**
+   * Changes the color of the referenced tree. Internally, a pre-defined array of colors is used which is
+   * why this function uses a colorIndex (between 0 and 500) instead of a proper color.
+   *
+   * @example
+   * api.tracing.setTreeColorIndex(3, 10);
+   */
+  setTreeColorIndex(treeId?: number, colorIndex: number) {
```

Comment:
Does it make sense to have the first parameter as optional? Calling the function with only one parameter will produce an incorrect result.

Reply:
Yeah, that's right. I can add it to `treeId: ?number` to avoid confusion (even with `treeId?` one would have to call `setTreeColorIndex(undefined, 3)` to satisfy the interface).

Reply:
You could also switch the order of parameters